packages/x-components/src/components/layouts/multi-column-max-width-layout.vue

```diff
@@ -410,9 +414,20 @@
     &__scroll-to-top {
       //layout
       grid-area: scroll-to-top;
-      margin-block-end: var(--x-size-margin-bottom-layout-columns-scroll-to-top, 10px);
-      margin-inline-start: var(--x-size-margin-left-layout-columns-scroll-to-top, 10px);
+      justify-content: flex-end;
+      width: 100%;
+      display: flex;
       z-index: 1;
+      pointer-events: none;
```

Comment:
This will not work, you are not able to click the scroll to top an emit the events.

Reply:
fixed